rust/src/core.rs

```diff
@@ -224,6 +229,45 @@ pub(crate) fn verify_kernel_hmac(rootfs: i32, moddir: &str) -> CxxResult<()> {
     verify_kernel_hmac_impl(&moddir).map_err(Into::into)
 }
 
+pub(crate) fn stage_container_rpms(rpms: Vec<String>) -> CxxResult<Vec<String>> {
```

Comment:
I'm fine with this as is, but: why not switch to doing this in the daemon path too?

Reply:
Yeah, maybe.

The idea with this is that it acts like the pkgcache repo in the container case where we don't have one. So we could use this client-side too, but then turn around and immediately import them into the pkgcache? I could see some utility to that. But I was actually thinking of going the other way and eventually drop this once we can use bare-split-xattrs more easily. That opens the door to using the full core assembly code. So then e.g. things like kernel replacement would re-use the same path.